package.json

```diff
@@ -12,28 +12,23 @@
     "vsce": "1.29.0"
   },
   "scripts": {
-    "postinstall": "lerna bootstrap",
-    "bootstrap": "lerna bootstrap",
+    "postinstall": "lerna bootstrap -- --no-package-lock",
```

Comment:
With Node 8 (or Yarn), there is a lock file that is created after `npm install`. The problem with this lock file is that it won't contain packages that are symlinked locally by Lerna. Thus, when we run `npm --prune production` it will also use the lock file and prune the symlinked @salesforce packages even though they are required. Thus, we tell Node 8 to not produce lock files during installation.

However, running other commands that affect the node_modules directory, e.g., `npm prune` will also affect the lock file. Since, we don't commit the lock file, this is fine.

Reply:
@vazexqi should we disable the file completely?

Like this:
`echo 'package-lock=false' >> .npmrc`

Reply:
I fee like doing it explicitly through Lerna's `--no-package-lock` is a bit more explicit, for now. I might change my mind about this later if there package-lock.json annoy me.